app/views/content/international-teacher-training-to-english-standards-introducing-iqts.md

```diff
@@ -0,0 +1,177 @@
+---
+title: International qualified teacher status (iQTS)
+description: |-
+ International qualified teacher status (iQTS) is a new teaching qualification backed by England’s Department for Education (DfE). iQTS is equivalent to qualified teacher status (QTS). You can train where you live and work. 
+date: "2021-08-06"
+image: "media/images/content/hero-images/0003.jpg"
+backlink: "../"
+keywords: 
+    - international
+    - QTS
+    - iQTS
+    - Qualified teacher status
+---
+
+## Introducing iQTS
+
+International qualified teacher status (iQTS) is a new teaching qualification backed by England’s Department for Education (DfE). iQTS is equivalent to qualified teacher status (QTS). You can train where you live and work. 
+
+## Key facts
+
+1. Apply now for courses starting September 2022
+2. Train for a year via remote learning and at a school in your region 
+3. Fees for iQTS vary – check with participating iQTS training providers
+4. Gain a professional teaching qualification recognised as equivalent to QTS
+
+## Six reasons to apply for iQTS
+
+1. iQTS leads to the award of English qualified teacher status ([QTS](https://www.gov.uk/guidance/qualified-teacher-status-qts)) with no further fees or training, and qualifies you to teach in all schools in England
+
+
+2. iQTS is open to international candidates from anywhere in the world who meet the eligibility criteria
+
+
+3. iQTS training and assessment are completed in your home country or school workplace, with no need to visit the UK
+
+
+4. iQTS offers high-quality, evidence-based training delivered by world class English teacher training providers, with in-school support from a trained mentor
```

Comment:
Sentence feels a little wordy. Maybe split it up or remove some information? Could be just:
'iQTS offers high-quality, evidence-based training with in-school support from a trained mentor.'